blog/2023/2023-06-21-open-source-101-a-beginners-guide-to-getting-started.md

```diff
@@ -52,7 +52,7 @@ If there aren’t any labels for beginners, don’t forget that the [best first
  
 > **Check out [OpenSauced Community Health Files](https://github.com/open-sauced/.github/tree/main).**
 
-- Engage with the Community. Join the project’s communication channels, like Slack or [Discord](https://discord.gg/6KYErCXf), their discussions, their blog, and introduce yourself. Ask questions and express your interest in contributing. Becoming a part of the community will expand your connections, help you to gain insights into ongoing discussions and development timelines, and may be an opportunity for mentorship from experienced contributors.
+- Engage with the Community. Join the project’s communication channels, like Slack or Discord, or [discussion forum](https://github.com/orgs/open-sauced/discussions), their discussions, their blog, and introduce yourself. Ask questions and express your interest in contributing. Becoming a part of the community will expand your connections, help you to gain insights into ongoing discussions and development timelines, and may be an opportunity for mentorship from experienced contributors.
```

Comment:
Because there's a discussion forum, should we omit the "their discussions" here and adjust it a bit?


```suggestion
- Engage with the Community. Join the project’s communication channels, like Slack or Discord, [discussion forum](https://github.com/orgs/open-sauced/discussions), or their blog, and introduce yourself. Ask questions and express your interest in contributing. Becoming a part of the community will expand your connections, help you to gain insights into ongoing discussions and development timelines, and may be an opportunity for mentorship from experienced contributors.
```